### PR TITLE
AP-1231 PHP wrapper update

### PR DESCRIPTION
**Pull request by rajatsib**
- Support for batch send functionality 
- Documentation update for sort parameter default usage
- Replaced html entities with quotes in docs/Model
- Updated version and improvement in useragent logic

### CODE REVIEW

File: lib/Model/SendSmtpEmailReplyTo.php [hunk 1]
```diff
@@ -36,7 +36,7 @@
  * SendSmtpEmailReplyTo Class Doc Comment
  *
  * @category Class
- * @description Email (required), along with name (optional), on which transactional mail recipients will be able to reply back. For example, {&#39;email&#39;:&#39;ann6533@example.com&#39;, &#39;name&#39;:&#39;Ann&#39;}.
+ * @description Email (required), along with name (optional), on which transactional mail recipients will be able to reply back. For example, {\&quot;email\&quot;:\&quot;ann6533@example.com\&quot;, \&quot;name\&quot;:\&quot;Ann\&quot;}.
```
Comment (shubhamUpadhyayInBlue) [inline lib/Model/SendSmtpEmailReplyTo.php:39]: `\&quot;` needs to be replaced

Comment (rajatsib) [reply lib/Model/SendSmtpEmailReplyTo.php]: updated

**Review by shubhamUpadhyayInBlue — APPROVED** [review]
LGTM

File: lib/Model/SendSmtpEmailReplyTo.php (hunk 1 above)
Comment (NicoleS021) [reply lib/Model/SendSmtpEmailReplyTo.php]: [](https://github.com/sendinblue/APIv3-php-library/commit/0b86513957d22c3fcd0598f5c961be2513f6c8d6)> `\&quot;` needs to be replaced

Comment (NicoleS021) [reply lib/Model/SendSmtpEmailReplyTo.php]: > [](https://github.com/sendinblue/APIv3-php-library/commit/0b86513957d22c3fcd0598f5c961be2513f6c8d6)> `\&quot;` needs to be replaced
> 
>

**Pull request merged**